docs/enhancements/0001_project_enhancements.md

```diff
@@ -0,0 +1,103 @@
+important that potential contributors be able to quickly setup their development environment, understand the 
+engagement model for development, and be able to discover the right forums for their area of interest.
+
+This document proposes that now is the right time to take these steps in order to facilitate objectives in 2023.
+
+### Goals
+
+1. Agree on the need for an official enhancement process
+2. Agree on the need to split the workloads and control plane areas of interest
+3. Agree on the need to create a formal definition of done that includes documentation and testing requirements
+4. Agree on the need to identify graduation criteria of existing APIs
+5. Assign ownership of the above for definition and implementation
+
+### Non-Goals
+
+1. Actually define the specifics of each of the goals - this should be up to the community
+
+## Proposal
+
+### Implement an enhancement proposal process
+
+An enhancement process has many benefits for a project, including:
+* creating an approchable method to driving change in a project
+* offering a forum for asynchronous feedback
+* providing a discoverable history of project change and the motivations behind the changes
+* setting expectations on the bar for contribution in areas like quality and graduation criteria
+
+This proposal suggests that kcp adopt an enhancement process that follows practices familiar to many in the 
+community via the [KEP process](https://github.com/kubernetes/enhancements/blob/adae507daeb490cfeb7f4d520d3d711362090c45/keps/NNNN-kep-template/README.md) 
+by tailoring aspects of that process to fit the kcp project's needs. This would include:
+
+1. Definition of an enhancment proposal template
+2. Creation of an enhancement proposal repository and removal of this proposal from the main repo to that repository.
+3. Establishing the approval criteria for said process and the expectations for review and feedback in a way that can
+be adopted by sub-projects identified later in this proposal.
+   
+This definition for enhancements should help establish the definition of done for an enhancement that includes any
+documentation and testing requirements, helping establish clear expectations for authors as well as helping the 
+community (as consumers of kcp) approach new features uniformly. 
+   
+### Split the workloads project from the control plane
+
+Within kcp there exists the following components that can be thought of as individual investment areas. 
+
+1. kcp - the generic control plane
+2. kcp tenancy - the workspaces and API pub/sub components
+3. kcp workloads - the transparent multi-cluster components
+4. [kcp edge workloads](https://github.com/kcp-dev/edge-mc) - the transparent multi-cluster components focused on edge use cases
+5. kcp controllers dev - the components focused on tooling and development of kcp aware controllers
+6. kcp catalog - discover component for published APIs
+
```

Comment:
Do we already have a clear dependency model in the code organization? e.g. "kcp tenancy" may have dependencies on kcp but not the other way around?

Reply:
I think we'll have to tease that apart a bit. @sttts mentions it in his comments and I'll update the proposal, but the most likely first step here is just split out workloads and leave kcp with api bindings and tenancy apis.  There is no urgency to separate those yet

Reply:
kcp-core <- kcp-tenancy
kcp-core <- kcp-tmc

These are clear. Whether we also need `kcp-tenancy <- kcp-tmc` we will see. At the moment I think it is not connected.